autoload/go/lint.vim

```diff
@@ -137,6 +141,12 @@ function! go#lint#Vet(bang, ...) abort
     let l:errorformat = "%-Gexit status %\\d%\\+," . &errorformat
     call go#list#ParseFormat(l:listtype, l:errorformat, out, "GoVet")
     let l:errors = go#list#Get(l:listtype)
+
+    if empty(l:errors)
+      call go#util#EchoError(l:out)
+      return
+    endif
```

Comment:
I might be mistaken, but I think this isn't needed. Did you add for a specific reason?

Reply:
Hmm I believe invocations errors that occur in `:GoVet` will silently fail without this block.

Try running `:GoVet -tags sometag -tags sometag`. `vim-go` will just report 
`vim-go: calling vet...` and not report the error that `vet` would (`go vet: tags flag may be set only once`)

`go#lint#ErrCheck` does something similar to report invocation errors.